web/.env.example

```diff
@@ -0,0 +1,3 @@
+CLIENT_ID=xx
+CLIENT_SECRET=xx
+IDP_DISCOVERY_URL=xx
```

Comment:
Could we re-order these settings to the config.py? Introducing another file for configurations sounds confusing to me.... Also working with .env's will introduce another project dependency.

Reply:
@P-T-I , I have removed the dep to dotenv and added these as settings to the configuration. I have also updated the doc.

Reply:
Awesome!